packages/bundlers/default/src/DefaultBundler.js

```diff
@@ -1036,15 +1036,21 @@ function createIdealGraph(
   }
 
   let modifiedSourceBundles = new Set();
-
   // Step Remove Shared Bundles: Remove shared bundles from bundle groups that hit the parallel request limit.
   for (let bundleGroupId of bundleGraph.getNodeIdsConnectedFrom(rootNodeId)) {
     // Find shared bundles in this bundle group.
     let bundleId = bundleGroupId;
 
     // We should include "bundle reuse" as shared bundles that may be removed but the bundle itself would have to be retained
     let bundleIdsInGroup = getBundlesForBundleGroup(bundleId); //get all bundlegrups this bundle is an ancestor of
-    if (bundleIdsInGroup.length > config.maxParallelRequests) {
+
+    // Filter out inline assests as they should not contribute to PRL
+    let numBundlesContributingToPLR = bundleIdsInGroup.reduce((count, b) => {
```

Comment:
I would just modify bundleIdsInGroup instead of using a new variable. (Unless we using bundleIdsInGroup somewhere else where you haven't replaced it with `numBundlesContributingToPLR` )

Reply:
good call!